data/block/block.go

```diff
@@ -18,7 +18,7 @@ import (
 // The body can be transmitted even before having built the heder and go through a prevalidation of each transaction
 
 // Type identifies the type of the block
-type Type uint8
+type Type uint32
```

Comment:
Why do we need uint32? Do you think that we can have more than 256 miniblock types?